src/components/Item.jsx

```diff
@@ -51,13 +54,43 @@ const Item = ({
     return estimatedInterval;
   };
 
+  function handleClick(e) {
+    e.preventDefault();
+    const confirmation = window.confirm(
+      `Are you sure you want to delete ${name}?`,
+    );
+
+    if (confirmation) {
+      db.collection('items')
+        .doc(id)
+        .delete()
+        .then(() => {
+          setActionMessage(`${name} successfully deleted!`);
+          setTimeout(() => {
+            setActionMessage(null);
+          }, 10000);
+        })
+        .catch((error) => {
+          console.error('Error removing document: ', error);
+        });
+    }
+  }
+
   const checked = checkDate(lastPurchasedDate);
   const className = checked ? 'checked' : '';
```

Comment:
This variable name has changed in the other pairs code, so we should probably change it here before we merge to (it's not checkedClassName I think) 😊